packaging/debian-unstable/patches/disable-seccomp-tests.patch

```diff
@@ -0,0 +1,29 @@
+Description: disable cmd/snap-seccomp tests
+ The snap-seccomp tests require a fork of the golang/x/net library to run.
+ Disable the tests until the necessary changes are upstream.
+Author: Michael Hudson-Doyle <michael.hudson@ubuntu.com>
+Origin: vendor
+Forwarded: not-needed
+Last-Update: 2017-08-15
+---
+This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
+--- a/cmd/snap-seccomp/export_test.go
++++ b/cmd/snap-seccomp/export_test.go
+@@ -1,5 +1,7 @@
+ // -*- Mode: Go; indent-tabs-mode: t -*-
+ 
++// +build ignore
```

Comment:
Same.. instead of patching this in, we can have a build tag that is only used when building for debian, and have that in the actual tree instead of patching in.

That said, why is this being disabled?  I don't think we're forking x/net.. our vendor file explicitly points towards the real repository.

Reply:
The fork is called github.com/mvo5/net/bpf. This delta will no longer apply to master when  https://github.com/snapcore/snapd/pull/3779 gets merged so perhaps we can just wait a day or so.